src/Service/Azure.DataApiBuilder.Service.csproj

```diff
@@ -48,6 +48,7 @@
 
 
     <ItemGroup>
+        <PackageReference Include="Azure.Identity" />
```

Comment:
This project doesnt depend on `Data.SqlClient`. Should we still include Azure.Identity explicitly here?

Reply:
Yes. Chatted offline.